bin/git-theta-filter

```diff
@@ -66,17 +71,15 @@ def smudge(args):
     repo = git_utils.get_git_repo()
     staged_file = file_io.load_staged_file(sys.stdin)
 
-    model_dict = defaultdict(dict)
-    for leaf_dir, keys in checkpoints.iterate_dir_leaves(
+    model_dict = {}
+    for keys, param_dir in utils.walk_parameter_dir(
```

Comment:
I think `utils.walk_parameter_dir` just returns the nested dict with leaves corresponding to `param_dir`s. I think this should iterate over `utils.flatten(utils.walk_parameter_dir(...))` and the body of the loop will need to be like main.

Reply:
Good catch, fixed.

Reply:
Whoops, forgot the `items` in my comment. `utils.flatten(utils.walk_parameter_dir(...)).items()`